### ISSUE TITLE
manual/scalaGuide/main/http/_Sidebar.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#1012]
- https://github.com/playframework-ja/translation-project/tree/2.3.x/manual/scalaGuide/main/http/_Sidebar.md
- diffs between [2.2.0/manual/scalaGuide/main/http/_Sidebar.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/scalaGuide/main/http/_Sidebar.md) and [2.3.5/manual/scalaGuide/main/http/_Sidebar.md](https://github.com/playframework-ja/translation-project/tree/2.3.x/manual/scalaGuide/main/http/_Sidebar.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/scalaGuide/main/http/_Sidebar.md    2013-07-29 23:52:24.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/scalaGuide/main/http/_Sidebar.md   2014-07-28 12:26:38.000000000 +0900
@@ -1,3 +1,4 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 ### HTTP programming

 - [[Actions, Controllers and Results | ScalaActions]]
@@ -23,12 +24,13 @@
 - [[Integrating with Akka | ScalaAkka]]
 - [[Internationalization | ScalaI18N]]
 - [[The application Global object | ScalaGlobal]]
-- [[Testing your application | ScalaTest]]
+- [[Testing your application | ScalaTestingYourApplication]]
+- [[Logging|ScalaLogging]]

 ### Advanced topics

 - [[Handling data streams reactively | Iteratees]]
-
-### Tutorials
-
-- [[Your first application | ScalaTodoList]]
\ No newline at end of file
+- [[HTTP API | HttpApi]]
+- [[HTTP Filters | ScalaHttpFilters]]
+- [[Dependency Injection | ScalaDependencyInjection]]
+- [[Extending Play|ScalaPlugins]]

```

**Comment by garbagetown** [comment]
やります :hand:

**Comment by garbagetown** [comment]
merged at #1059
